src/stories/baseComponents/utils-default-traj.tsx

```diff
@@ -0,0 +1,7901 @@
+import { Conflict, Trajectory } from '../../robot-trajectory-manager';
```

Comment:
Can you provide instructions to generate this file? Also would be good to add a comment at the top to indicate that this file is generated.

Reply:
Right now I have the script inside my own github. Would it necessary to create one under `osrf`?